### ISSUE TITLE
<wlr/backend/libinput.h> error when building from source

### ISSUE THREAD

**Issue by sdubinsky** [swaywm/sway#8351]
I am following the instructions on the wiki for building sway from source: https://github.com/swaywm/sway/wiki/Development-Setup

When I pull wlroots in as a subproject, sway builds and runs successfully.

When I install wlroots separately, sway fails to build.  I get the following build error:
```
ninja: Entering directory `build/'
[67/310] Compiling C object sway/sway.p/input_input-manager.c.o
FAILED: sway/sway.p/input_input-manager.c.o 
cc -Isway/sway.p -Isway -I../sway -Iinclude -I../include -Iprotocols -I/usr/include/cairo -I/usr/include/freetype2 -I/usr/include/libxml2 -I/usr/include/libpng16 -I/usr/include/harfbuzz -I/usr/include/glib-2.0 -I/usr/lib64/glib-2.0/include -I/usr/include/sysprof-6 -I/usr/include/pixman-1 -I/usr/include/libdrm -I/usr/include/json-c -I/usr/include/libevdev-1.0 -I/usr/include/pango-1.0 -I/usr/include/libmount -I/usr/include/blkid -I/usr/include/fribidi -I/usr/local/include/wlroots-0.19 -fdiagnostics-color=always -D_FILE_OFFSET_BITS=64 -Wall -Winvalid-pch -Wextra -Werror -std=c11 -O0 -g -DWLR_USE_UNSTABLE -D_POSIX_C_SOURCE=200809L -Wno-unused-parameter -Wno-unused-result -Wno-missing-braces -Wno-format-zero-length -Wundef -Wvla '-DSYSCONFDIR="//usr/local/etc"' '-DSWAY_VERSION="1.10-dev-f957c7e6 (" __DATE__ ", branch '"'"'master'"'"')"' -fmacro-prefix-map=../= -pthread -DWITH_GZFILEOP -MD -MQ sway/sway.p/input_input-manager.c.o -MF sway/sway.p/input_input-manager.c.o.d -o sway/sway.p/input_input-manager.c.o -c ../sway/input/input-manager.c
../sway/input/input-manager.c:7:10: fatal error: wlr/backend/libinput.h: No such file or directory
    7 | #include <wlr/backend/libinput.h>
      |          ^~~~~~~~~~~~~~~~~~~~~~~~
compilation terminated.
[80/310] Compiling C object sway/sway.p/input_cursor.c.o
ninja: build stopped: subcommand failed.
```

I noticed that `#include <wlr/backend/libinput.h>` appears twice in https://github.com/swaywm/sway/blob/f957c7e658871c27935f88f6e1d18b9db67f3808/sway/input/input-manager.c#L7, once in a guard(L7) once not (L26).

I have libinput-devel installed on my system, and when I configure wlroots it detects it but the config summary still says it's disabled, not sure if that's related:

```
...
Run-time dependency libinput found: YES 1.26.2
...
wlroots 0.19.0-dev

    drm-backend     : NO
    x11-backend     : YES
    libinput-backend: NO
    xwayland        : NO
    gles2-renderer  : YES
    vulkan-renderer : NO
    gbm-allocator   : YES
    session         : NO
    color-management: YES
    xcb-errors      : NO
    egl             : YES
    libliftoff      : NO

  Subprojects
    seatd           : NO Neither a subproject directory nor a seatd.wrap file was found.
```

To sum up:
1. There is a behavior difference between installing wlroots as a subproject and installing it system-wide
2. input-manager.c includes libinput.h twice
3. wlroots doesn't detect that libinput exists.  I mention this only as a possible symptom of a misconfiguration.

**Comment by emersion** [comment]
Sounds like a mistake. Would you be willing to send a patch to remove the duplicated, unguarded include?

**Comment by sdubinsky** [comment]
Sure!

**Comment by sdubinsky** [comment]
https://github.com/swaywm/sway/pull/8352

**Comment by sdubinsky** [comment]
Adding an additional note for anyone in the future that if you aren't seeing libinput backend support for wlroots it's because you don't have libseatd installed.
